src/main/java/org/broadinstitute/dsm/model/filter/participant/BaseFilterParticipantList.java

```diff
@@ -114,12 +114,8 @@ public BaseFilterParticipantList(String jsonBody) {
     }
 
     public void addParticipantDataConditionsToQuery(Map<String, Integer> allIdsForParticipantDataFiltering, Map<String, String> queryConditions, int filtersLength) {
-        if (allIdsForParticipantDataFiltering.isEmpty()) {
-            queryConditions.put(ElasticSearchUtil.ES, ElasticSearchUtil.BY_PROFILE_GUID + ElasticSearchUtil.EMPTY);
-        } else {
-            String newCondition = createNewConditionByIds(allIdsForParticipantDataFiltering, filtersLength);
-            queryConditions.merge(ElasticSearchUtil.ES, newCondition, (prev, next) -> prev + next);
-        }
+        String newCondition = createNewConditionByIds(allIdsForParticipantDataFiltering, filtersLength);
+        queryConditions.merge(ElasticSearchUtil.ES, newCondition, (prev, next) -> prev + next);
```

Comment:
concatenating previous and current value without some separator is okay? suffix of `newCondition` has separator already or maybe it does not need to be separated at all?

Reply:
" AND " is separator which is prefix of each new condition